packages/apollos-ui-kit/src/theme/types/light.js

```diff
@@ -18,8 +18,8 @@ const light = ({ barStyle, colors, alpha }) => ({
       inactive: colors.lightTertiary,
     },
     shadows: {
-      default: Color(colors.darkTertiary)
-        .fade(alpha.low)
+      default: Color(colors.black)
+        .fade(0.06)
```

Comment:
If your looking to have and end transparency of 10% than this would work and use our current theme values.
```suggestion
        .fade(alpha.high)
```

Reply:
Ok. Definitely didn't understand .fade like I though then :) That's ok! Appreciate it dude.

Reply:
You'll want to accept that commit suggestion and then run `yarn test -- -- -u` from the root directory in core to update snapshots. Then look over the changes to make sure you are seeing the correct values. Should just be a change to something like `rgba(0,0,0,0.1)`.